src/fn/closures/closure_examples/iter_find.md

```diff
@@ -1 +1,66 @@
-# Searching through iterators
+# Tìm kiếm thông qua các iterator (trình vòng lặp)
+
+`Iterator::find` là một hàm lặp qua một trình vòng lặp rồi tìm kiếm giá trị đầu tiên thỏa mãn một số điều kiện nào đó. Nếu không có giá trị nào thỏa mãn điều kiện, nó sẽ trả về  `None`. Điểm nhấn của nó:
+
+```rust
+pub trait Iterator {
+    // Kiểu lặp đi lặp lại.
+    type Item;
+
+    // `find` truyền `&mut self` nghĩa là người gọi có thể mượn và sửa đổi,
+    // nhưng không được tiêu thụ.
+    fn find<P>(&mut self, predicate: P) -> Option<Self::Item> where
+        // `FnMut` meaning any captured variable may at most be modified, not consumed.
+        // `FnMut` có nghĩa là bất kì biến nào được chụp nhiều nhất có thể được sửa đổi, không được sử dụng. 
```

Comment:
thiếu phần `&Self::Item` states it takes arguments to the closure by reference -> `&Self::Item` cho biết cho biết nó nhận các đối số vào closure bằng tham chiếu